src/Tizen.NUI/src/internal/WebBackForwardList.cs

```diff
@@ -0,0 +1,165 @@
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebBackForwardListItem : Disposable
+    {
+        /// <summary>
+        /// Get uri.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Uri Url
+        {
+            get
+            {
+                return new Uri(Interop.WebBackForwardListItem.GetUrl(SwigCPtr));
+            }
+        }
+
+        /// <summary>
+        /// Get title.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Title
+        {
+            get
+            {
+                return Interop.WebBackForwardListItem.GetTitle(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Get original url.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Uri OriginalUrl
+        {
+            get
+            {
+                return new Uri(Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr));
+            }
+        }
+
+        internal WebBackForwardListItem(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        internal static global::System.Runtime.InteropServices.HandleRef getCPtr(WebBackForwardListItem obj)
+        {
+            return (obj == null) ? new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero) : obj.SwigCPtr;
+        }
+
+        /// <summary>
+        /// Dispose for IDisposable pattern
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (type == DisposeTypes.Explicit)
+            {
+                //Called by User
+                //Release your own managed resources here.
+                //You should release all of your own disposable objects here.
+            }
+
+            base.Dispose(type);
+        }
+    }
+
+    /// <summary>
+    /// WebBackForwardList is a class for back-forward list of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebBackForwardList : Disposable
+    {
+        /// <summary>
+        /// Get item count.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int ItemCount
+        {
+            get
+            {
+                return Interop.WebBackForwardList.GetItemCount(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Get current item.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebBackForwardListItem GetCurrentItem()
+        {
+            System.IntPtr itemPtr = Interop.WebBackForwardList.GetCurrentItem(SwigCPtr);
+            return new WebBackForwardListItem(itemPtr, true);
```

Comment:
If the c reference of `WebBackForwardListItem` is managed in C++ side, the flag should be false, conceptually.

Reply:
Done.